tests/sentry/api/endpoints/test_organization_invite_request_index.py

```diff
@@ -93,6 +95,15 @@ def setUp(self):
 
     def test_simple(self):
         self.login_as(user=self.user)
+
+        with exempt_from_silo_limits():
+            assert (
+                RegionOutbox.objects.filter(
+                    category=OutboxCategory.ORGANIZATION_MEMBER_UPDATE
+                ).count()
+                == 0
+            )
+
         with self.tasks():
```

Comment:
If you want to also test the side effect of actually processing the outbox, you can change this line to `with self.tasks(), outbox_runner():` and it will process outboxes at the end of the context.